From 21afffb2ff68ea4ce9e1c82b845ea9b27ec0688a Mon Sep 17 00:00:00 2001
From: Marcos Bento <Marcos.Bento@ecmwf.int>
Date: Wed, 31 May 2023 11:01:33 +0200
Subject: [PATCH 3/3] Use MSG_NOSIGNAL only when defined ECFLOW-1881

---
 Http/src/ApiV1Impl.hpp | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/Http/src/ApiV1Impl.hpp b/Http/src/ApiV1Impl.hpp
index f9fdde32a..9190a4e68 100644
--- a/Http/src/ApiV1Impl.hpp
+++ b/Http/src/ApiV1Impl.hpp
@@ -17,8 +17,17 @@
 //
 /////////1/////////2/////////3/////////4/////////5/////////6/////////7/////////8
 
+#include <sys/socket.h>
+#if defined(MSG_NOSIGNAL)
+// The MSG_NOSIGNAL flag should be defined by <sys/socket.h>
+// (as per POSIX: https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/functions/send.html),
+// but that is not always the case for OSX.
+//
+// If MSG_NOSIGNAL is not defined, we leave it up to httplib to use default flags.
+//
 #define CPPHTTPLIB_SEND_FLAGS MSG_NOSIGNAL
 #define CPPHTTPLIB_RECV_FLAGS MSG_NOSIGNAL
+#endif
 
 #ifdef ECF_OPENSSL
     #define CPPHTTPLIB_OPENSSL_SUPPORT
-- 
2.32.0 (Apple Git-132)
